github_utils.py

```diff
@@ -578,3 +578,70 @@ def mark_commit_status(commit, repository, context="default", state="pending", u
         params['context'] = s['context']
         github_api('/repos/%s/statuses/%s' % (repository, commit), params=params)
   return
+
+def get_branch(repository, branch_name):
+    get_gh_token(repository)
+    data = github_api("/repos/%s/branches/%s" % (repository, branch_name), method="GET")
+    return data
+
+
+def get_git_tag(repository, tag_name):
+    get_gh_token(repository)
+    data = github_api(
+        "/repos/%s/git/ref/tags/%s" % (repository, tag_name), method="GET"
+    )
+    return data
+
+
+def create_git_tag(
+    repository, tag_name, commit_sha, tagger_name, tagger_email, tag_date=None
+):
+    get_gh_token(repository)
+    if tag_date is None:
+        tag_date = datetime.datetime.now()
+    tag_date_s = tag_date.replace(microsecond=0).isoformat()
+    params = {
+        "tag": tag_name,
+        "message": "Create tag %s" % tag_name,
+        "object": commit_sha,
+        "type": "commit",
+        "tagger": {"name": tagger_name, "email": tagger_email, "date": tag_date_s},
```

Comment:
`tagger` is optional , so so not explicitly pass it. The github token should should be enough to provide the needed information to github. So please remove tagger_name , tagger_email and drop `"tagger"` from the cal